config.json

```diff
@@ -898,6 +898,18 @@
         "move_semantics"
       ]
     },
+    {
+      "slug": "fizzy",
+      "uuid": "6b209749-d4af-45c2-bbdc-27603ce6979f",
+      "core": false,
+      "unlocked_by": "luhn",
+      "difficulty": 7,
```

Comment:
Note on positioning: the Rust track currently has three exercises other than `fizzy` with the "generics" topic. None of them are core exercises. The easiest of these is a difficulty 4, unlocked by "luhn", so I chose to make this a peer exercise.

I don't believe that it's worth worrying too much about positioning, because Rust is currently in the middle of the track reordering project, and we can expect the position of this exercise to shift, possibly substantially.